src/controllers/hello-world.ts

```diff
@@ -1,7 +1,11 @@
 import {api} from '@loopback/core';
+import {createEmptyApiSpec, OpenApiSpec} from '@loopback/openapi-spec';
 import {apiSpec} from './hello-world.api';
 
-@api(apiSpec)
+var openApiSpec: OpenApiSpec = createEmptyApiSpec();
+Object.assign(openApiSpec, apiSpec);
```

Comment:
Why is this needed? Is some code modifying our OpenApiSpec object? IIRC, we are always creating a clone of the spec objects to avoid the need to copy them. 

If your goal is to add fields from empty api spec, then please note that `@api` decorator no longer requires a full Swagger preamble, it takes a smaller object with two properties only: `{basePath?, paths}`.